docs/io.cozy.sessions.md

```diff
@@ -0,0 +1,27 @@
+- [Cozy sessions doctype](#cozy-sessions-doctype)
+  - [`io.cozy.sessions`](#iocozysessions)
+  - [Example](#example)
+
+# Cozy sessions doctype
+
+## `io.cozy.sessions`
+
+The `io.cozy.sessions` doctype holds the web sessions informations.
+
+- `instance` {object}: The instance of the sessions
```

Comment:
I think you should remove the `instance` field: it is used purely inside the stack (in memory), but not persisted to CouchDB or accessible by the clients.